docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +173,81 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+* **Alerts related by session ID**: Shows alerts that generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. 
++
+NOTE: To use this feature, you must enable the **Include session data** setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+* **Alerts related by process ancestry**: Shows alerts that are related by process events on the same linear branch. When you examine this information further, note that alerts generated from processes on child or related branches are not shown. 
++
+NOTE: To access data about alerts related by process ancestry, you must have a https://www.elastic.co/pricing[Platinum or higher subscription].
+
+[discrete]
+[[prevalence-section]]
+=== Prevalence
+
+The Prevalence sub-section shows how often data from the alert was observed on other host and user events. Prevalence calculations use data from the alert's highlighted fields and examine user and host events that occurred during the last 30 days. Click **Prevalence** to further examine prevalence data in the <<prevalence-sub-tab,Prevalence sub-tab>>. 
+
+[discrete]
+[[prevalence-sub-tab]]
+==== Prevalence sub-tab
+
+The **Prevalence** sub-tab is located under the **Insights** tab in the left panel. It shows relationships with other alerts, events, users, and hosts. Update the date time picker for the table to show data from a different time range. 
 
 [role="screenshot"]
-image::images/threat-intel-sub-sec.png[Enriched with Threat Intelligence subsection with matched file hash, 700]
+image::images/prevalence-sub-tab-lp.png[Table within the Prevalence sub-tab]
 
-TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+The following details are provided:
+
+* **Field**: Shows <<investigation-section,highlighted fields>> for the alert and any custom highlighted fields that were added to the alert's rule.
+* **Value**: Shows values for highlighted fields and any custom highlighted fields that were added to the alert's rule.
+* **Alert count**: Shows the total number of alert documents that have identical field values, including the alert you're currently examining. For example, if the `host.name` field alert count is 5, there are five total alerts with the same `host.name` field value.
++
+NOTE: Alert documents contain the {ecs-ref}/ecs-allowed-values-event-kind.html#ecs-event-kind-signal[`event.kind:signal`] field-value pair.  
+* **Document count**: Shows the total number of event documents that have identical field values. A dash (`——`) displays if there are no event documents that match the field value.
++
+NOTE: Event documents contain the {ecs-ref}/ecs-allowed-values-event-kind.html#ecs-event-kind-event[`event.kind:event`] field-value pair.  
```

Comment:
`event.kind:event` is not correct here. This columns shows all the documents where `event.kind` is NOT `signal`.
There are plenty of `event.kind` values other than `event`. These are all the values that I'm aware of:
- alert
- asset
- enrichment
- event
- metric
- state
- pipeline_error
- signal

So while the alerts count column shows all the documents with `event.kind: signal`, the documents count columns show all the documents with `event.kind` NOT `signal`

Reply:
@PhilippeOberti thanks for clarifying this. My notes show that Event count column queries documents that contain `event.kind` NOT `signal`, but I wasn't clear whether that meant everything _except for_ `event.kind:signal` or if was only event docs that specifically contained `event.kind:event`. If it's the former, I can update the description and note as follows:

> * **Document count**: Shows the total number of event documents that have identical field values. A dash (`——`) displays if there are no documents that match the field value.
> +
> NOTE: The Document count column only retrieves documents that _don't_ contain the {ecs-ref}/ecs-allowed-values-event-kind.html#ecs-event-kind-signal[`event.kind:signal`] field-value pair.  

I'd also update the note for the Alert document column to be:

> NOTE: The Alert document column only retrieves documents that contain the {ecs-ref}/ecs-allowed-values-event-kind.html#ecs-event-kind-signal[`event.kind:signal`] field-value pair.

Reply:
yup it is the former :)

Reply:
Resolving in next batch of updates.